### PR TITLE
refactor: remove components from local uic if matching base uic

### PR DESCRIPTION
**Pull request by emilyjablonski**
# Pull Request Template

## Issue Overview

This PR addresses #3283

- [x] This change addresses only certain aspects of the issue

## Description

This PR renames Detroit's local UIC to detroit-ui-components. For components where the Detroit version differs, all pages have been changed to pull in a local import for that component from this newly named folder. For components where the Detroit version was the same (or almost the same / the UIC version had improvements Detroit did not), the imports remain the same `@bloom-housing/ui-components` but now it is technically pulling in a "different" version from the base UIC.

Removed local Detroit component, now pulling in the UIC version
* ButtonGroup
* ExpandableText
* LocalizedLink
* Video
* Card
* DashBlock
* HousingCounselor
* InfoCard
* MediaCard
* StatusBar
* StatusItem
* ExygyFooter
* SiteFooter
* DOBField
* DateField
* Dropzone
* Form
* HouseholdMemberForm
* PhoneField
* Select
* Textarea
* TimeField
* All helpers/
* HeaderBadge
* PreferencesList
* Breadcrumbs
* TabNav
* Tabs
* AlertNotice
* ApplicationStatus
* ErrorMessage
* SiteAlert
* StatusAside
* StatusMessage
* Overlay
* ApplicationTimeline
* FormForgotPassword
* ContentAccordion
* ListingMap
* ListingsGroup
* UnitTables
* Contact
* ContactAddress
* GetApplication
* ListingUpdated
* NumberedHeader
* OrDivider
* QuantityRowSection
* ReferralApplication
* SubmitApplication
* DownloadLotteryResults
* EventSection
* FormSignIn
* ForSignInAddPhone
* FormSignInErrorBox
* FormSignInMFACode
* ResendConfirmationModal
* All prototypes/
* ContentSection
* FooterSection
* ListSection
* MarkdownSection
* ResponsiveContentList
* ResponsiveWrappers
* CategoryTable
* MinimalTable
* StackedTable
* Message

Remaining discrepancies (most are very small), kept locally
* Button
* ExpandableContent
* LinkButton
* ActionBlock
* FormCard
* ImageCard
* Tooltip
* ViewItem
* Field
* FieldGroup
* AgPagination
* Heading
* HeadingGroup
* Hero
* PageHeader
* SiteHeader
* StepHeader
* Icon
* FooterNav
* LanguageNav
* ProgressNav
* AlertBox
* Drawer
* LoadingOverlay
* Modal
* AdditionalFees
* ListingCard
* ListingDetailHeader
* ListingDetails
* ListingDividerLine
* WhatToExpect
* FormTerms
* GridSection
* InfoCardGrid
* AgTable
* GroupedTable
* StandardTable
* Description
* Tag

## How Can This Be Tested/Reviewed?

There will be some visual changes. Ideally, they are only minimal and are improvements. Some of the components that Detroit is now using from UIC were actually a little different, but the UIC version is better or more consistent, so this PR and dev are not identical.

[Here](https://docs.google.com/document/d/1CixA6lJdnuqQX0TkmgrVPW_DOnGhoe_I7Wo4HYRXENA/edit?usp=sharing) are some known and IMO acceptable or preferred visual changes.

QA could be relatively large. We need to audit both sites and compare to dev to make sure there are no unwanted or breaking visual changes.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [x] I have exported any new pieces added to ui-components
- [x] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

**Comment by emilyjablonski** [comment]
For some reason this is looking super different and mildly broken on the deploy previews vs locally. Gonna investigate that before review.

**Review by ludtkemorgan — APPROVED** [review]
I haven't found any visual issues. Only thing I've noticed is the README.md at the root directory should probably update the `ui-components` description and point it to the pulled out ui-component. 

Also unrelated to this pr I noticed the README also references housingbayarea. We should probably re-look at what is in the docs and make sure it's relevant for Detroit

File: sites/partners/styles/overrides.scss [hunk 1]
```diff
@@ -59,3 +59,13 @@
     }
   }
 }
+
+.alert-box.primary {
+  background-color: var(--bloom-color-gray-400);
+}
+
+#sign-in {
```
Comment (emilyjablonski) [inline sites/partners/styles/overrides.scss:67]: This is a Big Sad but this has caused a lot of spinning and is blocking other work

**Comment by ColinBuyck** [comment]
I'm noticing that a small "0" is appearing on some listings and I couldn't seem to reproduce on dev. 
Preview: https://deploy-preview-1542--detroit-public-dev.netlify.app/listing/52616882-c06c-4230-8592-7f93f93e0880/655_hazelwood_655_hazelwood_st_detroit_mi
Dev: https://detroit-public-dev.netlify.app/listing/52616882-c06c-4230-8592-7f93f93e0880/655_hazelwood_655_hazelwood_st_detroit_mi
Screenshot 2023-02-07 at 4 58 04 PM

**Comment by ColinBuyck** [comment]
So very small but noticed that the color is off for this icon in the Community Types and Photos section
Screenshot 2023-02-07 at 5 30 00 PM
Screenshot 2023-02-07 at 5 30 57 PM

**Comment by ColinBuyck** [comment]
Sorry, but even smaller... it seems like the add listing button is 1 px different between local and dev.
Screenshot 2023-02-07 at 5 41 36 PM

**Comment by emilyjablonski** [comment]
Ty for the feedback @ColinBuyck! I fixed #1 and #2, #3 w the buttons is known and will make us more consistent w UIC

**Comment by ludtkemorgan** [comment]
@emilyjablonski Thanks for making those changes. I'm not seeing the first two issues Colin mentioned so I'm ok to merge with just 1 approval. However, we don't have the ability to override the 2 approvers. At least I don't in Detroit. We can get someone else to click the approve button. Also, did you have a chance to look at the README https://github.com/CityOfDetroit/bloom/pull/1542#pullrequestreview-1286202908 ?

Your commit messages are great for this PR `fix: why`, `fix: why did this work??`

**Review by ColinBuyck — APPROVED** [review]
@emilyjablonski  Sorry I missed that 3rd piece in your description! I'm only able to see the recent code changes since I'm away from my computer but they look good and I feel comfortable approving based on that🦕

**Comment by emilyjablonski** [comment]
@ludtkemorgan I was hoping to do that in a follow-up since a fair amount is looking outdated but can also do this here! Open either way

**Comment by ludtkemorgan** [comment]
@emilyjablonski That sounds like a great plan to do it separately since it needs a bit of work.

**Pull request merged**